### ISSUE TITLE
Totally lost

### ISSUE THREAD

**Issue by AnnMajeske** [githubschool/funny-lemur#162]
I missed one step at the beginning and I'm totally lost.  How do I create AnnMajeske/github-games?

**Comment by AnnMajeske** [comment]
@crichID See my original comment - totally lost

**Comment by crichID** [comment]
@AnnMajeske to create your copy of the repository, you will want to go to https://github.com/githubschool/github-games and click the fork button in the top right corner. While you are doing that, I will grab the instructions for you to catch up!

**Comment by crichID** [comment]
Here are the instructions: https://github.github.io/training-manual/#_update_readme

**Comment by dunbam01** [comment]
Same here....   missed something early on, and could not listen to lecture while trying to catch up...  now lost.

**Comment by crichID** [comment]
@dunbam01 the good news is that you can drop the earlier activities and just start where @beardofedu is now!

**Comment by crichID** [comment]
@AnnMajeske @dunbam01 FYI, all of the steps for the activities are in the training manual and will also be in the video, so please don't be discouraged - Git is hard the first time :wink:

**Comment by AnnMajeske** [comment]
Why it git so hard?  If you're starting from scratch creating a tool you should make it intuitive not hard to use!  One thing I'm noticing is that the current branch names are the same as the file names which makes the class hard to follow.

**Comment by AnnMajeske** [comment]
@crichID after this tutorial I would not use git if I had a choice.  Unfortunately I do not have a choice.  This is going to be even harder on a large (millions of lines of code) project.

**Comment by crichID** [comment]
@AnnMajeske I know Git is tricky and you aren't the first to say it :wink:. Don't give up ... we use the command line because we know that people who understand the fundamentals have little problem switching to a GUI, but there are also some very good GUIs out there (e.g. GitHub Desktop and GitKraken) and you can avoid the command line most of the time.

Git does an exceptional job on huge projects. It was created to handle the Linux Kernel which is way up in the millions. I encourage you to keep your commits and pull requests small and focused and keep trying -- you will feel comfortable very soon!

**Comment by AnnMajeske** [comment]
@crichID I'm much more comfortable on the command line than the GUI.  It is the switching back and forth  and knowing the right thing to click on on the GUI when all you have are little icons that I don't understand in different places.  You can't always keep commits and pull requests small when you're working on giant projects, that's the point.  You also need many collaborative branches that might be around a long time for another and git is designed for short term, small, branches.

**Comment by crichID** [comment]
@AnnMajeske we work with a lot of people who use long running release branches or use a development branch as the default and then protect who can merge to master. There are many workflow options with Git so I hope you can find one that is great for you and your team.
